float32 float_0to1
---
bool is_smaller